service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/mapper/StructuredObservationValueMapper.java

```diff
@@ -40,6 +46,31 @@ public String mapObservationValueToStructuredElement(IBaseElement value) {
             .apply(value);
     }
 
+    public String mapInterpretation(CodeableConcept interpretation) {
+        Coding coding = interpretation.getCodingFirstRep();
+        String code = coding.getCode();
+
+        switch (code) {
+            case "H":
+            case "HH":
+            case "HU":
+                return String.format(INTERPRETATION_CODE_TEMPLATE, "HI", "2.16.840.1.113883.2.1.6.5",
```

Comment:
Looks like the string "2.16.840.1.113883.2.1.6.5" is common across all three cases. 
Could move this into the template string, instead of formatting in?

Reply:
Agree with your comment & have added this in 3baa038